docs/general-development/excel-services-known-issues-and-tips.md

```diff
@@ -75,7 +76,7 @@ For example, if a cell contains characters with hexadecimal values 0x1, 0x2 ...
   
     
     
- **System.InvalidOperationException: Client found response content type of 'text/html; charset=utf-8', but expected 'text/xml'. The request failed with the error message: -- <html> <head> <title>' ', hexadecimal value 0x01, is an invalid character.</title>**
+ **System.InvalidOperationException: Client found response content type of 'text/html; charset=utf-8', but expected 'text/xml'. The request failed with the error message: -- &lt;html&gt; &lt;head&gt; &lt;title&gt;' ', hexadecimal value 0x01, is an invalid character.</title>**

```

Comment:
Correct markdown escaping of angled brackets should be prefixed with\; don't change to the HTML encoding.